### PR TITLE
Multiple versions of PHP now work on Apache

### PR DESCRIPTION
**Pull request by xushangning**
Confirmed to work on my own machine with a fresh install of Laravel 4.2 and PHP 5.6. See [laravel/homesteadv7.11.0](https://github.com/laravel/homestead/releases/tag/v7.11.0) and [v7.14.2](https://github.com/laravel/homestead/releases/tag/v7.14.2) for more details.

### CODE REVIEW

**Comment by taylorotwell** [comment]
Hmm, I don't see where we said it **wasn't** supported before?

File: homestead.md [hunk 1]
```diff
@@ -477,9 +477,7 @@ After running the command, you will see an Ngrok screen appear which contains th
 <a name="multiple-php-versions"></a>
 ### Multiple PHP Versions
 
-> {note} This feature is only compatible with Nginx.
```
Comment (xushangning) [inline homestead.md]: Here it is.

Comment (xushangning) [reply homestead.md]: @taylorotwell

**Comment by browner12** [comment]
Yah, I would just get rid of line 480, and leave the rest as is. Unless we can start versioning the Homestead documentation, we're just kind of always assuming latest release.

**Comment by xushangning** [comment]
I see. I will open another pull request.

**Pull request closed without merging**